src/legacy/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx

```diff
@@ -265,6 +263,8 @@ export default function TransactionConfirmationModal({
 }: ConfirmationModalProps) {
   const { chainId } = useWalletInfo()
   const setShowFollowPendingTxPopup = useUpdateAtom(handleFollowPendingTxPopupAtom)
+  const activities = useMultipleActivityDescriptors({ chainId, ids: [hash || ''] }) || []
```

Comment:
Should we change this method to return `[]` by default instead of null, so we don't have to do `|| []` ?

Reply:
Fixed. Thanks!